projects/etos_suite_runner/src/etos_suite_runner/lib/esr_parameters.py

```diff
@@ -58,7 +55,35 @@ def get_status(self) -> dict:
             return self.environment_status.copy()
 
     @property
-    def artifact_created(self) -> EiffelArtifactCreatedEvent:
+    def testrun_id(self) -> str:
+        """Testrun ID returns the ID of a testrun, either from a TERCC or environment."""
+        if self.etos.config.get("testrun_id") is None:
+            if os.getenv("IDENTIFIER") is not None:
+                self.etos.config.set("testrun_id", os.getenv("IDENTIFIER", "Unknown"))
+            else:
+                self.etos.config.set("testrun_id", self.tercc.meta.event_id)
+        testrun_id = self.etos.config.get("testrun_id")
+        if testrun_id is None:
+            raise TypeError("Testrun ID is not set, neither in TERCC nor IDENTIFIER "
+                            "environment variables")
+        return testrun_id
+
+    @property
+    def iut_id(self) -> str:
```

Comment:
Will there be more ids here? Can these two methods be refactored to a generic method, such as:

self.get_id("testrun_id", "IDENTIFIER", self.tercc.meta.event_id)